specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1770,6 +1887,130 @@
+                    "readOnly": true,
+                    "items": {
+                        "$ref": "#/definitions/TopologyResource"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "TopologyResource": {
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "readOnly": true,
+                    "$ref": "#/definitions/TopologyResourceProperties"
+                }
+            },
+            "allOf": [{
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "$ref": "#/definitions/Location"
+                }
+            ]
+        },
+        "TopologyResourceProperties": {
+            "type": "object",
+            "properties": {
+                "calculatedDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "readOnly": true,
+                    "description": "The UTC time on which the topology was calculated"
+                },
+                "resources": {
+                    "type": "array",
+                    "readOnly": true,
+                    "description": "Azure resources which are part of this topology resource",
+                    "items": {
+                        "$ref": "#/definitions/TopologySingleResource"
+                    }
+                }
+            }
+        },
+        "TopologySingleResource": {
+            "type": "object",
+            "properties": {
+                "resourceId": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "Azure resource id"
+                },
+                "severity": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "The security severity of the resource"
+                },
+                "recommendationsExist": {
+                    "type": "boolean",
+                    "readOnly": true,
+                    "description": "Indicates if the resource has security recommendations"
+                },
+                "internetFacing": {
+                    "type": "boolean",
+                    "readOnly": true,
+                    "description": "Indicates if the resource has direct connectivity to the Internet"
+                },
+                "topologyScore": {
+                    "type": "integer",
+                    "readOnly": true,
+                    "description": "Security score of the resource based on its severity, siblings security status and location in topology (to be used for clustering similar objects together)"

```

Comment:
can you change the term security score in the description as well?
I think that "Score of the resource based on its security severity" is good enough